config/rbac/role.yaml

```diff
@@ -29,10 +29,3 @@ rules:
   - get
   - patch
   - update
```

Comment:
Are these removed by accident?

Reply:
looks like it. curious why it is generated this way. will remove it.

Reply:
I meant we should add them back. They were added via PR: https://github.com/kubernetes-sigs/application/pull/155.

Reply:
restored.